internal/module/openapi.go

```diff
@@ -43,7 +44,7 @@ func helmFormatModuleImages(m *Module, rawValues map[string]any) (chartutil.Valu
 	vers = append(vers, "autoscaling.k8s.io/v1/VerticalPodAutoscaler")
 	caps.APIVersions = vers
 
-	digests, err := GetModulesImagesDigests(m.GetPath())
+	digests, err := GetModulesImagesDigests(ToLowerCamel(m.GetName()), m.GetPath())
```

Comment:
Why is ToLowerCamel here?

Reply:
Because when we use such names as "ingress-nginx"  as module name (or Chart name) it converts to lower camel case in deckhouse controller

Reply:
We take the name of the module from its definition
https://github.com/deckhouse/dmt/blob/main/internal/module/module.go#L124
Thus, I do not see much point in transformation.
This is not critical, but so, thoughts out loud.